examples/simulation/Tutorial1/src/FairFastSimExample2.cxx

```diff
@@ -182,6 +175,15 @@ FairTutorialDet1Point* FairFastSimExample2::AddHit(Int_t trackID,
     return new (clref[size]) FairTutorialDet1Point(trackID, detID, pos, mom, time, length, eLoss);
 }
 
+Bool_t FairFastSimExample2::CheckIfSensitive(std::string name)
```

Comment:
```suggestion
Bool_t FairFastSimExample2::CheckIfSensitive(const std::string& name)
{
  return name == "fast_sim_vol_n2";
}
```

Reply:
The problem is that the function API is quite difficult to change. It is being used by many experiments already. 
The guts of the function is a different thing - I do agree that the std::string is nowadays better than the TString.

Reply:
:(, I see, then at least change the implementation.
Or perhaps keep the bad one and add new one, and add `__attribute__((deprecated("Use Bool_t FairFastSimExample2::CheckIfSensitive(const std::string& name)")))` to the old signature. Then at least people will see a warning and hopefully change it. But because it is overloaded through inheritance, maybe the warning will not be triggered.